### PR TITLE
Add to getting-started -> framework-deploy

### PR DESCRIPTION
**Pull request by ghost**
Published. https://kdf.csco.cloud/getting-started/mac/#step-6-framework-deploy

### CODE REVIEW

**Review by rtluckie — APPROVED** [review]

File: content/getting-started/readfiles/framework-deploy-main.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+The `.envrc` file in this directory is a symlink to `../common/dot.envrc`. The `dot.envrc` file behind the symlink
+configures your shell for the current cluster directory name. In this case, that means `docker-for-desktop` 
+
+- `README.md`
+
+The `README.md` in this directory contains guidance specific to the `docker-for-desktop` Kubernetes cluster.
+
+- `mh/main.yaml`
+
+The [mh](https://github.com/cisco-sso/mh) tool which ships with the KDK container iamge is capable of managing one or
```
Comment (dcwangmit01) [inline content/getting-started/readfiles/framework-deploy-main.md]: TYPO: iamge -> image

**Review by dcwangmit01 — APPROVED** [review]
Awesome you kept this dry.

File: content/getting-started/readfiles/framework-deploy-main.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+.  ..  .envrc  .git  .gitignore  LICENSE  README.md  apps  bin  clusters  ensure-env.sh
+```
+
+- `.envrc`
+
+Upon entering a directory that contains an `.envrc` file, the `direnv` tool installed in the KDK container will
+automatically source the `./.envrc` file if it is present.
+
+For example, in the framework-deploy pattern, `direnv` tunes your shell's environment variables for each cluster
+directory into which you change (`cd`).
```
Comment (dcwangmit01) [inline content/getting-started/readfiles/framework-deploy-main.md]: Does it make sense to add a sentence:

"It is through these environment variables that all of the tools are configured (e.g. kubectl, docker, helm, etc)

Comment (ghost) [reply content/getting-started/readfiles/framework-deploy-main.md]: Added. Thanks!

File: content/getting-started/readfiles/framework-deploy-main.md (hunk 1 above)
Comment (ghost) [reply content/getting-started/readfiles/framework-deploy-main.md]: Fixed. Thanks!

**Comment by ghost** [comment]
@dcwangmit01  please take another look to confirm my fixes and merge at will. Thanks!

**Pull request merged**